src/makeCalculator.js

```diff
@@ -36,8 +36,41 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(value, number) {
+      value(number);
+
+      return this;
+    },
+
+    add(figure) {
+      calculator.result += figure;
+    },
+
+    subtract(figure) {
+      calculator.result -= figure;
+    },
+
+    multiply(figure) {
+      calculator.result *= figure;
+    },
+
+    divide(figure) {
+      calculator.result /= figure;
+    },
```

Comment:
consider case when figure is 0 and return an error, because we can't divide to 0